.github/workflows/test.yml

```diff
@@ -43,10 +43,11 @@ jobs:
 
     services:
       postgres:
-        image: postgres:11.6-alpine
```

Comment:
Can you take these out? They should be reviewed and tested seperately.

Reply:
yes that was a test, the migrations weren't running properly